### ISSUE TITLE
PA Review: News Feed

### ISSUE THREAD

**Issue by trwong** [trwong/fakebook#3]
- [x] Backend: DB, model, controller, views
- [x] Redux loop: ajax, actions, reducers
- [x] Presentational Components
- [x] Styling
- [x] Smooth, bug-free navigation
- [x] Adequate and appropriate seeds

**Comment by trwong** [comment]
commenting on a post will come with future mvp: "Comments/posting on each others’ walls"

**Comment by ajansel** [comment]
This looks great! I only noticed a few things: 
- You will likely want the nav bar to stay positioned at the top and the feed scroll beneath it so the user can always access it (this would be the case for profile and news feed) 
- Also I mentioned the font sizing on my comments in the profile feedback I gave. That would also be relevant here as the font sizing seems to differ across some posts. 
- The user should also be able to edit and delete the posts the make on their wall / other peoples' wall 
- You will won't to filter your posts to not include sensitive or inappropriate language / topics
